### DISCUSSION TITLE
Add ability to give a superset role access to all schema's under a particular database

### DISCUSSION THREAD

**Discussion by mdeshmu** [apache/superset#21473] [Ideas]
Currently there isn't a way to give a role access to all the schema's under one particular database something like `all schema access on [database]`. Please add support for such permission grants. 
We don't want to give all_datasource_access as it is too open and will give access to data sources from other databases as well. 
We also do not want to add schema access on [database].[schema] for every schema as our list is huge. 

Here is the link to slack discussion: https://apache-superset.slack.com/archives/C016B3LG5B4/p1662993356593079

**Comment by rusackas** [comment]
This isn't a bug, per se, so I'll migrate the Issue to a Discussion. @eschutho or @john-bodley may be able to relay it to appropriate parties.

**Reply by eschutho** [reply]
also cc @dpgaspar

**Reply by mdeshmu** [reply]
@dpgaspar can this feature be  added in 2.1.0 please?
